sentry-ruby/spec/sentry/transport/http_transport_rate_limiting_spec.rb

```diff
@@ -0,0 +1,252 @@
+require 'spec_helper'
+
+RSpec.describe "rate limiting" do
```

Comment:
@jan-auer if you have time, can you go through this spec file to see if all the cases are described correctly? thanks

Reply:
Thank you, the specs all look good. Added two comments for additional specs below.